app/frontend/src/styles/application.scss

```diff
@@ -66,21 +66,18 @@ body {
   }
 
   .govuk-main-wrapper {
-    @include dashboard;
-  }
+    background-color: govuk-colour('light-grey');
+    margin-bottom: govuk-spacing(6);
+    padding-bottom: 0;
+    padding-top: 0;
 
-  &.vacancies_index,
-  &.vacancies_show,
-  &.home_index {
-    .govuk-main-wrapper {
-      > .govuk-width-container {
-        margin-bottom: govuk-spacing(6);
-        margin-top: 0;
-
-        > .govuk-grid-row {
-          margin-top: govuk-spacing(4);
-        }
-      }
+    .content-wrapper {
+      background-color: govuk-colour('white');
+    }
+
+    h1,
```

Comment:
I think the wrapper should have a margin at the top since h1 isn't always the top element in the page